contrib/clojure-package/examples/infer/imageclassifier/test/infer/imageclassifier_example_test.clj

```diff
@@ -46,24 +46,16 @@
         predictions (classify-single-image classifier image-file)]
     (is (some? predictions))
     (is (= 5 (count predictions)))
-    (is (every? #(= 2 (count %)) predictions))
-    (is (every? #(string? (first %)) predictions))
-    (is (every? #(float? (second %)) predictions))
-    (is (every? #(< 0 (second %) 1) predictions))
-    (is (= ["n02123159 tiger cat"
-            "n02124075 Egyptian cat"
-            "n02123045 tabby, tabby cat"
-            "n02127052 lynx, catamount"
-            "n02128757 snow leopard, ounce, Panthera uncia"]
-           (map first predictions)))))
+    (is (= 5 (count predictions)))
```

Comment:
duplicate?

Reply:
In this test, it's testing the 3 signatures of the call (1 new one was added with the FLOAT64 PR to add dtype)

```clojure
predictions-all (infer/classify-image classifier image)
predictions-with-default-dtype (infer/classify-image classifier image 10)
predictions (infer/classify-image classifier image 5 dtype/FLOAT32)
```
with

```
(is (= 1000 (count predictions-all)))
(is (= 10 (count predictions-with-default-dtype)))
(is (= 5 (count predictions)))
```

Reply:
oh my bad - I was looking at the other test not in the examples directory! It is a duplicate - I'll update :)